gc/base/MemoryPoolAddressOrderedList.cpp

```diff
@@ -587,9 +610,52 @@ MM_MemoryPoolAddressOrderedList::collectorAllocate(MM_EnvironmentBase *env, MM_A
 	return addr;
 }
 
+/* Check the free entry being used and align it to appropriate boundaries for TLH allocation as required by GC.
+ * SATB Sweep has strict requirements for TLH to not span multiple Sweep Chunks. */
+bool
+MM_MemoryPoolAddressOrderedList::alignTLHForGC(MM_EnvironmentBase *env,  MM_HeapLinkedFreeHeader *freeEntry, uintptr_t *consumedSize)
```

Comment:
We could add a lightweight hint what imposes the aliment (but at the same time not be too specific like sweep/compact)... something like alignTLHForParallelGC 

Even comment could be more generic:  Some GCs  during some phases mey require that TLHs not span heap chunks for parallel processing.

Move the comment into hpp